Pollo/ViewControllers/PollsViewController+Extension.swift

```diff
@@ -163,10 +168,10 @@ extension PollsViewController: SliderBarDelegate {
     func scrollToIndex(index: Int) {
         let indexPath = IndexPath(item: 0, section: index)
         pollsCollectionView.scrollToItem(at: indexPath, at: [], animated: true)
-    }
 
-    func changeNewGroupButton(status: Bool) {
-        newGroupButton.isHidden = status
+        self.bottomPaddingView.backgroundColor = index == 1 ? .lightGrey : .darkestGrey
+        self.joinSessionContainerView.isHidden = index == 1
+        self.createSessionContainerView.isHidden = index == 0
```

Comment:
can we make `index == 0` and `index == 1` into variables which describe what they represent?

Reply:
Yup!